discovery-provider/scripts/up.sh

```diff
@@ -58,6 +58,7 @@ fi
 (
     # mv ./node_modules away, temporarily
     cd ${PROTOCOL_DIR}/discovery-provider/es-indexer
+    npm i
```

Comment:
Thanks for catching this! Fwiw, I didn't see this earlier in the previous version of [service-commands](https://github.com/AudiusProject/audius-protocol/commit/aa2f65a9119d86a3f7db44c0fa0d294878968d7f#diff-a88334f7ae0bc40f3ab3982d5f623c1d3bac1e9e1422e4ff2a23d5f69384b8afL191). I bring this up in case it is an indicator for another fix that's needed.

Would we also need this somewhere after this line in the [Dockerfile](https://github.com/AudiusProject/audius-protocol/blob/138398afb2042eda4c7257212118fe0b4cc50b97/discovery-provider/Dockerfile#L82)?